app/src/main/java/com/lastaoutdoor/lasta/data/db/DatabaseManager.kt

```diff
@@ -3,13 +3,10 @@ package com.lastaoutdoor.lasta.data.db
 import com.google.firebase.Firebase
 import com.google.firebase.firestore.SetOptions
 import com.google.firebase.firestore.firestore
+import com.lastaoutdoor.lasta.data.model.profile.ActivitiesDatabaseType
 import com.lastaoutdoor.lasta.data.model.user.HikingLevel
 import com.lastaoutdoor.lasta.data.model.user.UserModel
 import com.lastaoutdoor.lasta.data.model.user.UserPreferences
-import java.util.Date
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.launch
 import kotlinx.coroutines.tasks.await
 
 private const val USERS_COLLECTION = "users"
```

Comment:
Same here, add this DB in string resources

Reply:
Thank you for the feedback. Actually, I haven't coded the class DatabaseManager myself, I only changed it in order to not have errors. This class in does not implement dagger-hilt hence being difficult to pass the application context as parameter to this class. In conclusion, I won't be changing it for now as we'll need to look further all the changes necessary to make this class work